docker/build/mule.go.debian.Dockerfile

```diff
@@ -4,11 +4,11 @@ FROM python:3.7
 COPY --from=docker /usr/local/bin/docker /usr/local/bin/docker
 COPY *.yaml /root/
 
-RUN pip install mulecli
+RUN apt-get update && apt-get install -y autoconf bsdmainutils build-essential curl git libboost-all-dev && \
+    curl https://dl.google.com/go/go1.12.linux-amd64.tar.gz | tar -xzf - && \
+    mv go /usr/local && \
+    pip install mulecli
```

Comment:
I'm only hardcoding the version of golang here since this will be changed very shortly to reference a single location that will download and install golang.

Reply:
I think this one is go 1.12.0, we may want to use g1.12.17 here instead. They're pretty consistent with their urls also so you could do something like this to make it configurable:

```
ARG GO_VERSION=1.12
RUN apt-get update && apt-get install -y autoconf bsdmainutils build-essential curl git libboost-all-dev && \
    curl https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -xzf - && \
    mv go /usr/local && \
    pip install mulecli
```